apps/backend/src/deposits/entities/cash-deposit.entity.ts

```diff
@@ -25,7 +25,7 @@ export class CashDepositEntity {
   program_code?: string;
 
   @Column({ type: 'date' })
-  deposit_date: string;
+  deposit_date: Date | string;
```

Comment:
I think this should be just `Date` and then be casted to string of typeorm needs it to be that way with a pre db hook or something similar.

Reply:
I might need your help on this. I've tried using "@BeforeInsert" hook from typeorm but I'm getting the same error :/

Reply:
I suppose we could switch to using a "timestamp" in the db vs "date"?

Reply:
This article explains the typeorm issue:

https://wanago.io/2021/03/15/postgresql-typeorm-date-time/